Src/BootCamp.Chapter/Inventory.cs

```diff
@@ -1,26 +1,47 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection.Metadata.Ecma335;
```

Comment:
Watch for unused usings - you shouldn't need this.

Reply:
Ouch, you're right, I must have added this accidentaly. I've removed all unused namespaces from all classes.